src/applications/claims-status/tests/components/claim-files-tab/AddFilesForm.unit.spec.jsx

```diff
@@ -64,22 +57,15 @@ describe('<AddFilesForm>', () => {
       expect($('.files-form-information', container)).to.exist;
     });
 
-    it('should show uploading modal', () => {
+    it('uploading modal should not be visible', () => {
       const { container } = render(
-        <Provider store={store}>
-          <AddFilesForm uploading {...fileFormProps} />,
-        </Provider>,
+        <AddFilesForm uploading {...fileFormProps} />,
```

Comment:
I would swap the order of these, otherwise the value of uploading may (or may not) be overridden by `fileFormProps` if it has a `uploading` key